src/Microsoft.SqlTools.ServiceLayer/TableDesigner/Contracts/Requests/ProcessTableDesignerEditRequest.cs

```diff
@@ -24,6 +25,8 @@ public class ProcessTableDesignerEditResponse
         public bool IsValid { get; set; }
 
         public TableDesignerIssue[] Issues { get; set; }
+
+        public Dictionary<string, string> PropertyBag { get; set; }
```

Comment:
`Metadata` seems like a better name to use (especially given that we use that same terminology elsewhere)